plugins/all-modules-page/src/test/kotlin/templates/MultiModuleDocumentationTest.kt

```diff
@@ -39,7 +39,7 @@ class MultiModuleDocumentationTest : MultiModuleAbstractTest() {
             includes = listOf(folder.root.resolve("README.md"))
         }
 
-        testFromData(configuration, preserveOutputLocation = true) {
+        testFromData(configuration) {
```

Comment:
That's the culprit, because the flag was set to `true` - it used the output location from configuration, which is the `out` folder by default